ui/lib/core/addon/components/search-select.js

```diff
@@ -136,7 +142,8 @@ export default Component.extend({
   handleChange() {
     if (this.selectedOptions.length && typeof this.selectedOptions.firstObject === 'object') {
       if (this.passObject) {
```

Comment:
Could we maybe move this if/else block into `configureObject`, which would simplify the logic within `handleChange`?

Reply:
Oh yeah - I'll take a look!